packages/iTwinUI-react/src/core/Header/HeaderDropdownButton.tsx

```diff
@@ -0,0 +1,73 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import { DropdownMenu } from '../DropdownMenu';
+import SvgCaretDownSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretDownSmall';
+import SvgCaretUpSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretUpSmall';
+import { PolymorphicForwardRefComponent, useTheme } from '../utils';
+import { DropdownButtonProps } from '../Buttons';
+import { useMergedRefs } from '../utils';
+
+export type HeaderDropdownButtonProps = DropdownButtonProps;
```

Comment:
`styleType` is a prop in `DropdownButtonProps` but nothing happens with it in `HeaderDropdownButton`. Is that ok?

Reply:
`styleType` is not needed for any of the `HeaderButton` types because they are always `borderless`